spec/buftok_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'filewatch/buftok'
+
+describe FileWatch::BufferedTokenizer do
+
+  context "when using the default delimiter" do
+    subject { FileWatch::BufferedTokenizer.new }
+
+    it "splits the lines correctly" do
+      message = "hello\nworld\n"
```

Comment:
Going even farther, I think this might be a good example for a shared_example as you're basically doing the same test, just changing the delimiter and the message.